### PR TITLE
Add info how to use this plugin

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,43 @@ IoniconsAPI plugin
...
 
+### Jelly
 ```xml
-<l:icon src="symbol-symbolName plugin-ionicons-api" />
+<l:icon src="symbol-symbolName-outline plugin-ionicons-api" />
+```
+
+### Groovy
+```groovy
+l.task(icon:"symbol-symbolName-outline plugin-ionicons-api")
```
Comment (timja) [inline README.md:34]: examples should probably match between groovy and jelly?

File: README.md [hunk 2]
```diff
@@ -10,11 +10,43 @@ IoniconsAPI plugin
...
+### Java
+
+```java
+@Override
+public String getIconClassName() {
+    return "symbol-symbolName-outline plugin-ionicons-api";
+}
+```
+
+Preferably, use symbols containing `-outline`, if available.
```
Comment (timja) [inline README.md:46]: add why

File: README.md [hunk 3]
```diff
@@ -10,11 +10,43 @@ IoniconsAPI plugin
 
 This plugin provides [ionicons](https://ionic.io/ionicons), also known as "symbols", for your Jenkins plugins.
 
+First, add the ionicons-api as dependency to your `pom.xml`:
+
+```xml
```
Comment (timja) [inline README.md:15]: worth linking to the equivalent of https://plugins.jenkins.io/slack/#dependencies?

It gives you a copy paste snippet with the latest version

I didn't link to the one for this as it doesn't appear to be on the plugin site yet

**Review by timja — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (NotMyFault) [reply README.md:15]: > worth linking to the equivalent of https://plugins.jenkins.io/slack/#dependencies?

Absolutely!

**Comment by NotMyFault** [comment]
Btw, did you get a chance to approve my request to install renovate here?

**Comment by timja** [comment]
> Btw, did you get a chance to approve my request to install renovate here?

there's no request that I can see

File: README.md [hunk 4]
```diff
@@ -10,11 +10,35 @@ IoniconsAPI plugin
 
 This plugin provides [ionicons](https://ionic.io/ionicons), also known as "symbols", for your Jenkins plugins.
 
+Add the ionicons-api as dependency to your `pom.xml`:
+
+Click [here](https://plugins.jenkins.io/ionicons-api/#dependencies) for a snippet.
```
Comment (timja) [inline README.md:15]:
```suggestion
See the [plugin site](https://plugins.jenkins.io/ionicons-api/#dependencies) for a snippet.
```

**Comment by NotMyFault** [comment]
> > Btw, did you get a chance to approve my request to install renovate here?
> 
> there's no request that I can see

Rip, requested one (again).

**Comment by timja** [comment]
approved it

**Pull request merged**